### PR TITLE
Add Rebuild support for currently loaded project/solution

### PR DESCRIPTION
**Pull request by kzu**
Shift+F6 will rebuild with the last saved arguments for the
currently open project/solution.

### CODE REVIEW

File: src/StructuredLogViewer/MainWindow.xaml [hunk 1]
```diff
@@ -19,6 +19,7 @@
     <Menu DockPanel.Dock="Top">
       <MenuItem Header="_File">
         <MenuItem Header="_Build Solution/Project..." Click="Build_Click" InputGestureText="F6" />
+        <MenuItem Header="Rebuild Solution/Project" Command="{Binding RebuildCommand}" InputGestureText="Shit+F6" />
```
Comment (KirillOsenkov) [inline src/StructuredLogViewer/MainWindow.xaml]: LOL what a typo! :)

Comment (KirillOsenkov) [inline src/StructuredLogViewer/MainWindow.xaml]: LOL what a typo! :)

Comment (KirillOsenkov) [inline src/StructuredLogViewer/MainWindow.xaml]: LOL what a typo! :)

Comment (KirillOsenkov) [inline src/StructuredLogViewer/MainWindow.xaml]: LOL what a typo! :)

Comment (KirillOsenkov) [inline src/StructuredLogViewer/MainWindow.xaml]: LOL what a typo! :)

Comment (kzu) [inline src/StructuredLogViewer/MainWindow.xaml]: LOL!!!! 😃😃😃

**Comment by kzu** [comment]
💩 fixed!

**Pull request merged**